code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm

```diff
@@ -93,3 +105,8 @@
 		datum_reference.qliphoth_change(-1)
 	return
 
+/mob/living/simple_animal/hostile/abnormality/judgement_bird/attackby(obj/item/I, mob/living/user, params)
```

Comment:
Exec won't activate this. Include same code in Bullet_Act so I can dual wield this man into a map nuke please.